ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -35,6 +35,9 @@ tests:
     from: src
 - as: verify
   commands: |
+    export GOLANGCI_LINT_VERSION=$(cat .golangciversion)
```

Comment:
@dave-tucker until we actually have a .golangciversion file in the CNO repo, just default to a reasonable version